apps/ui/src/Pages/RidesFeed/CreateRideDialog.tsx

```diff
@@ -65,6 +71,21 @@ const CreateRideDialog = ({
     setOpen(false);
   };
 
+  const handleChange = (fieldName: string, value: string) => {
+    const error = validateField(fieldName, value);
+    setValidationErrors((prev) => ({ ...prev, [fieldName]: error ? error : null }));
+    switch (fieldName) {
```

Comment:
move switch case to an object. then `fieldName` can be `fieldName: typeof keyof object` and you can use `object[fieldName]();` for better type safety

Reply:
@Kovler10 
Changed it to an object